### PR TITLE
Three tier checkout price cards

### PR DESCRIPTION
**Pull request by rBangay**
## What are you doing in this PR?

Adding the price cards component to the top of the three tier checkout (2nd page) if you are inside the three tier ab test variant and you have selected a one off contribution type.

The price cards replace the order summary component.

## Screenshots
Screenshot 2024-01-16 at 10 23 41

### CODE REVIEW

File: support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	margin: 6px 0 ${space[3]}px;
+	${from.desktop} {
+		margin-bottom: 0;
+	}
+`;
+
+const title = css`
+	${headline.xsmall({ fontWeight: 'bold' })}
+	${from.tablet} {
+		font-size: 28px;
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx:29]: Maybe worth a `${headline.small({ fontWeight: 'bold' })}` instead?

File: support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+				const destination = `/${countryGroups[countryGroupId].supportInternationalisationId}/contribute`;
+				navigateWithPageView(navigate, destination, abParticipations);
+			}}
+		>
+			Back
+		</Button>
+	);
+
+	return (
+		<div // this can be split out into it's own component called orderSummaryPriceCards
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx:66]: Remove comment now?

File: support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+					otherAmount,
+					currency,
+					paymentInterval,
+					onAmountChange,
+					minAmount,
+					onOtherAmountChange,
+					hideChooseYourAmount,
+					errors,
+				}) => (
+					<>
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx:90]: Is this empty tag necessary?

**Review by paul-daniel-dempsey — APPROVED** [review]
Few small tweaks and looks good to me. Price Cards display as expected.

File: support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx (hunk 3 above)
Comment (rBangay) [reply support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx:90]: nope, good spot 👍

File: support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx (hunk 1 above)
Comment (rBangay) [reply support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx:29]: the initial font size declaration `xsmall` is correct 24px ... as for the font-size changing from tablet breakpoints I don't think a proper way to do this has been decided while using source, with that in mind simply changing the font-size is the smallest change to the css we can make in order to make this change and avoid re-declaring duplicate css propertties

**Pull request merged**